src/api/appHub.ts

```diff
@@ -0,0 +1,61 @@
+import { D2ApiGeneric } from "./d2Api";
+import { D2ApiResponse } from "./common";
+
+/* https://docs.dhis2.org/master/en/developer/html/webapi_app_store.html */
+
+export class AppHub {
+    constructor(public d2Api: D2ApiGeneric) {}
+
+    list(): D2ApiResponse<App[]> {
+        return this.d2Api.get<App[]>("/appHub");
+    }
+
+    install(version: string): D2ApiResponse<void> {
+        return this.d2Api.post(`/appHub/${version}`);
+    }
+}
+
+export interface App {
+    appType: string;
+    created: Date;
+    description: string;
+    developer: AppDeveloper;
+    id: string;
+    images: AppImage[];
+    lastUpdated: Date;
```

Comment:
`created` and `lastUpdated` dates or strings? (there are some more in the types)